unlocks.csv

```diff
@@ -323,7 +323,7 @@ Analysis|PlanterShelf|0,0,PlanterShelf
 Analysis|PowerTransmitter|0,0,PowerTransmitter
 Analysis|PropulsionCannon|0,0,PropulsionCannon
 Analysis|Seaglide|0,0,Seaglide
-Analysis|Seamoth|0,1,Seamoth
+Analysis|Seamoth|0,0,Seamoth
```

Comment:
Flagged as non-standard just to reduce the gap between unlocks and blueprints to prevent 'default' behaviour on standard options. And shouldn't be one anyway, stupid Plasteel Ingot.